src/common/playleaderboard/leaderBoard.css

```diff
@@ -1,25 +1,25 @@
 .leaderboard-table-cell {
```

Comment:
Why this file change is required? Are you using the same formatter?

Reply:
Yes. I am using prettier @2.8.2 as code formatter. I will revert the mentioned changes . Thank you.